pages/index.html

```diff
@@ -4,9 +4,37 @@
   <meta charset="UTF-8">
```

Comment:
It is a convention to include index.html at the project root level and additional pages in a directory such as `pages`